### ISSUE TITLE
Apothecary suggestions

### ISSUE THREAD

**Issue by dimitre** [openframeworks/apothecary#451]
we can organize better the branches and releases, make them use the same name. 
update master with bleeding content and have a master release
I think for macos libraries there are too much complex libs in download 1 in comparision to 2 and 3. if it is better divided or even splitted in more files the actions could finish sooner.

**Comment by dimitre** [comment]
OF downloader should not use cache. I've noticed it avoided downloading latest apothecary libs even when they are different.

**Comment by danoli3** [comment]
Yep will have a better solution soon. byte size should never be the same for actual downloads though so must have been gremlins

Artifact cache is almost fixed now
